reinforcement-learning/README.md

```diff
@@ -0,0 +1,35 @@
+[![image](https://sigopt.com/static/img/SigOpt_logo_horiz.png?raw=true)](https://sigopt.com)
+
+# Tuning Deep Q-Networks With SigOpt
+
+This example uses SigOpt to tune a Deep Q-Network (DQN) to solve a reinforcement learning problem using OpenAi's gym simulation environments.
+
+## Setup
+
+1. `git clone https://github.com/sigopt/sigopt-examples.git`
+2. Install requirements `pip install -r requirements.txt`
+3. `cd sigopt-examples/reinforcement-learning`
+4. Add your SigOpt [API token](https://sigopt.com/docs/overview/authentication) to line 10 of `dqn.py`
+5. Execute the script by running: `python dqn.py`
```

Comment:
Maybe a sixth step saying you can check the progress of the experiment at https://sigopt.com/experiments ?